docs/reference/commandline/swarm_init.md

```diff
@@ -142,6 +143,11 @@ Format example is `--default-addr-pool-mask-length 24`
 
 This flag sets up task history retention limit.
 
+### `--data-path-port`
+
+This flag specifies data path port number.
+Format example is `--data-path-port 4789`
```

Comment:
This example isn't very useful, because this is the default port; also, we may want to show a full example, and show how to check the port that is used;

```
docker swarm init --advertise-addr=xxx --data-path-addr=yyy --data-path-port=zzz

# output of the command
```

Then check the output of `docker info` to verify;

```
docker info --format='{{.Something.Something}}'
4789
```